docs/standard/microservices-architecture/implement-resilient-applications/implement-custom-http-call-retries-exponential-backoff.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Implementing custom HTTP call retries with exponential backoff
-description: .NET Microservices Architecture for Containerized .NET Applications | Implementing custom HTTP call retries with exponential backoff
+title: Explore custom HTTP call retries with exponential backoff
+description: .NET Microservices Architecture for Containerized .NET Applications | Explore custom HTTP call retries with exponential backoff. The purpose of this secction is just to explain and explore the problem.
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 05/26/2017
+ms.date: 06/08/2018
 ---
-# Implementing custom HTTP call retries with exponential backoff
+# Explore custom HTTP call retries with exponential backoff
 
-In order to create resilient microservices, you need to handle possible HTTP failure scenarios. For that purpose, you could create your own implementation of retries with exponential backoff.
+In order to create resilient microservices, you would need to handle possible HTTP failure scenarios. For that purpose, you could create your own implementation of retries with exponential backoff.
 
-In addition to handling temporal resource unavailability, the exponential backoff also needs to take into account that the cloud provider might throttle availability of resources to prevent usage overload. For example, creating too many connection requests very quickly might be viewed as a Denial of Service ([DoS](https://en.wikipedia.org/wiki/Denial-of-service_attack)) attack by the cloud provider. As a result, you need to provide a mechanism to scale back connection requests when a capacity threshold has been encountered.
+**Important note:** The purpose of this section is just to show you how you could try to create your own custom code to implement HTTP call retries. However, it is not recommended to do it by your own but to use more powerful and reliable while simpler to use mechanisms, such as HttpClientFactory with Polly, available since .NET Core 2.1. Those recommended approaches are explained in the next sections. 
```

Comment:
section -> article?

Second sentence is a bit hard to parse. Suggestion:
However, we recommend that you use for your microservices more powerful and reliable yet simpler to use mechanisms, such as HttpClientFactory with Polly. The HttpClientFactory API is available since .NET Core 2.1.

use the traditional important notes for docs?

```markdown
> [!IMPORTANT]
> Text
```